File: Microservices/displayProducts.py
from flask import Flask
import mysql.connector

server_port = 8000
app = Flask(__name__)


@app.route('/displayProducts')
def displayProducts():
    # Creating connection object
    mydb = mysql.connector.connect(
        host = "localhost",
        user = "<your-db-username>",
        password = "<your-db-password>", 
        database = "Products"
    )

    print(mydb)
    
    html = "<br><table border='1'><th>Product </th><th>Description</th><th>Cost($)</th><th>Count</th>"

    cursor = mydb.cursor()
    cursor.execute("SELECT * FROM Footwear")
    for x in cursor:
        html += "<tr>"
        html +=  "<td>" + str(x[1]) + "</td>";
        html +=  "<td>" + str(x[2]) + "</td>";
        html +=  "<td>" + str(x[3]) + "</td>";
        html +=  "<td><select class='productCount' data-productname='" + str(x[1]) + "' data-productid=" +  str(x[0]) + " name='count'><option value=0>0</option><option value=1>1</option><option value=2>2</option></select></td>"
        html +=  "</tr>";

    return html


@app.route('/')
def index():
  return 'Home page.'


if __name__ == "__main__":
    app.run('0.0.0.0',port=server_port)

File: Microservices/displayProducts.php
<?php

$databaseServer = '<your-rds-endpoint>';
$username = '<your-db-username>';
$password = '<your-db-password>';
$database = "Products";
$port = '3306';

// Create connection
// PHP 8.1+ makes mysqli throw on failure (blank 500 page). Report it instead.
mysqli_report(MYSQLI_REPORT_OFF);
$conn = mysqli_connect($databaseServer, $username, $password, $database, $port);
if (!$conn) {
  http_response_code(500);
  echo "<h3>DisplayProducts: database connection failed</h3>";
  echo "<p>Host: <code>" . htmlspecialchars($databaseServer) . "</code></p>";
  echo "<p>MySQL says: <code>" . htmlspecialchars(mysqli_connect_error()) . "</code></p>";
  exit;
}
$sql = "SELECT * FROM Footwear";
// Query all products using SQL
$result = mysqli_query($conn, $sql);
$htmlString = "";
$htmlString .= 
$htmlString .=  "<br>";
$htmlString .=  "<table border='1'>";
$htmlString .=  "<th>Product </th><th>Description</th><th>Cost($)</th><th>Count</th>";
while ($row = mysqli_fetch_assoc($result))
{
    $htmlString .=  "<tr>";
    $htmlString .=  "<td>" . strval($row["Name"]) . "</td>";
    $htmlString .=  "<td>" . strval($row["Description"]) . "</td>";
    $htmlString .=  "<td>" . strval($row["Cost"]) . "</td>";
    $htmlString .=  "<td><select class='productCount' data-productname='".strval($row["Name"])."' data-productid=".strval($row["ItemID"])." name='count'><option value=0>0</option><option value=1>1</option><option value=2>2</option></select></td>";
    $htmlString .=  "</tr>";
}
$htmlString .=  "</table>";
echo $htmlString;
?>